app/reducers/lendableObjects.ts

```diff
@@ -0,0 +1,41 @@
+import { createSelector } from '@reduxjs/toolkit';
+import { LendableObject } from 'app/actions/ActionTypes';
+import type { RootState } from 'app/store/createRootReducer';
+import createEntityReducer from 'app/utils/createEntityReducer';
+import type { EntityId } from '@reduxjs/toolkit';
+
+export default createEntityReducer({
+  key: 'lendableObjects',
+  types: {
+    fetch: LendableObject.FETCH,
+    mutate: LendableObject.CREATE,
+    delete: LendableObject.DELETE,
+  },
+});
```

Comment:
It would be awesome if you used `legoAdapter` and `createSlice` instead of `createEntityReducer` here and in `lendingInstances`!
I am planning to remove `createEntityReducer` once I have refactored all the existing usages, so I don't really want any new usages:)